easybuild/easyconfigs/r/RAxML/RAxML-8.2.4-foss-2016a-hybrid-avx2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'RAxML'
+version = '8.2.4'
+versionsuffix ='-hybrid-avx2'
+
+homepage = 'https://github.com/stamatak/standard-RAxML'
+description = "RAxML search algorithm for maximum likelihood based inference of phylogenetic trees."
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+sources = ['v%(version)s.zip']
+source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
+
+buildopts = '-f Makefile.AVX2.HYBRID.gcc'
```

Comment:
@bedroge the existing easyconfig also has `CC="$CC"` in here, any reason you removed that?

It's probably not strictly need to have it here since `foss` uses `GCC` as compiler, but it's better to leave it there in case someone uses this easyconfig as a base for building with another (non-GCC) toolchain...